src/main/java/io/papermc/codebook/lvt/LvtAssignmentSuggester.java

```diff
@@ -478,4 +480,35 @@ private static boolean isStringAllUppercase(final String input) {
 
         return null;
     }
+
+    // 3 instructions, load "this" local var, getfield, return - TODO maybe if there is a CAST,
+    private static final IntPredicate[] OPCODES_IN_ORDER = new IntPredicate[] {
+        i -> i == Opcodes.ALOAD, i -> i == Opcodes.GETFIELD, i -> i >= Opcodes.IRETURN && i <= Opcodes.RETURN
```

Comment:
`ALOAD` should always be of slot `0` for `this`.

Reply:
Do you mean that I should be checking more than just the opcode in these predicates? that I should also check the slot of the var insn here?